nuxeo-coldstorage-web/i18n/messages-fr.json

```diff
@@ -22,11 +16,10 @@
   "documentContentView.restoreFromColdStorage.tooltip": "Restaurer depuis le stockage à froid",
   "documentContentView.restoreFromColdStorage.success": "Commencez à restaurer le contenu du document à partir du stockage à froid",
   "documentContentView.restoreFromColdStorage.error": "Erreur de restauration du contenu du document à partir du stockage à froid.",
-  "documentContentView.retrieveFromColdStorage.confirm" : "Confirmer",
-  "documentContentView.retrieveFromColdStorage.cancel" : "Annuler",
-  "documentContentView.retrieveFromColdStorage.ask.confirmation": "Récupérer ce document auprès du stockage à froid?",
-  "documentContentView.retrieveFromColdStorage.description": "Il en résultera un appel asynchrone et vous recevrez un courriel lorsque le document sera prêt.",
->>>>>>> NXP-30384: add manual translations for coldstorage restore feature
+  "documentContentView.retrieveFromColdStorage.confirm": "Confirmer",
```

Comment:
You have a merge's issue on this part. `>>>>>>> NXP-30384: add manual translations for coldstorage restore feature` `<<<<<<< HEAD`